spec/spec_helper.rb

```diff
@@ -0,0 +1,102 @@
+RSpec.configure do |config|
+
+  config.before(:all) do
+  FactoryBot.reload
```

Comment:
looks like this should be inside rails_helper
as we use factory_bot_rails, so we gonna test all factories per rails env

Reply:
Yes, you are right, I've fixed that.